legislation_analysis/utils/constants.py

```diff
@@ -24,8 +24,9 @@
 CONGRESS_DATA_FILE = os.path.join(
     API_DATA_PATH, "congress_abortion_legislation_full-text.csv"
 )
-CONGRESS_DATA_FILE_CLEANED = os.path.join(
-    CLEANED_DATA_PATH, "congress_legislation_cleaned.fea"
+CONGRESS_DATA_CLEANED_FILE_NAME = "congress_legislation_cleaned.fea"
+CONGRESS_DATA_CLEANED_FILE = os.path.join(
```

Comment:
`CONGRESS_DATA_FILE_CLEANED` was changed to `CONGRESS_DATA_CLEANED_FILE` so we could have constants sorted by their state (Cleaned, Clustered, etc.).